CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 CHANGELOG
 =========
 
+1.0.1 - 2024-02-06
+------------------
+
+* Add build_attrs filter to tailwind_filters.py
+* Add attrs.html template for tailwind layout
+* Refactor select.html and select_option.html templates
+* Add id attribute to select elements
+* Fix ID attribute in select element
+* Add tests for select attributes
+
```

Comment:
AutoPub will automatically update the `CHANGELOG.md` file, so there is no need to modify it manually.

Instead of manually editing the `CHANGELOG.md` file, add a `RELEASE.md` file containing:

```markdown
Release type: patch

* Add `build_attrs` filter to `tailwind_filters.py`
* Add `attrs.html` template for Tailwind layout
* Refactor `select.html` and `select_option.html` templates
* Add ID attribute to `select` elements
```

AutoPub will see the `patch` release type value in the `RELEASE.md` file and increment both the `pyproject.toml` and `crispy_tailwind/__init__.py` files from `1.0.0` to `1.0.1`.

Then, AutoPub will use the description of the changes in the `RELEASE.md` file and automatically update the `CHANGELOG.md` file.

Does that make sense? Anything I can clarify?

Reply:
👌